DbAdapterTestUtilities/Stubs/StubPrimaryKeyValue.cpp

```diff
@@ -26,15 +26,15 @@ namespace systelab::db::test_utility {
 				switch(fieldType)
 				{
 					case db::BOOLEAN:
-						fieldValue.reset( new StubFieldValue(field.getName(), field.getBooleanDefaultValue()) );
+						fieldValue.reset( new StubFieldValue(field.getName(), std::make_optional(field.getBooleanDefaultValue())) );
```

Comment:
That's what I was afraid of when I saw that you removed the non-optional constructors.

Reply:
Yep, looks odd. As discussed offline, we are gonna revert this